compiler/cli/cli-common/src/org/jetbrains/kotlin/cli/common/arguments/K2JSCompilerArguments.kt

```diff
@@ -25,11 +25,11 @@ class K2JSCompilerArguments : CommonCompilerArguments() {
     }
 
     @GradleOption(DefaultValues.StringNullDefault::class)
-    @Argument(value = "-output", valueDescription = "<path>", description = "Output file path")
+    @Argument(value = "-output", valueDescription = "<filepath>", description = "Destination *.js file for the compilation result")
     var outputFile: String? by NullableStringFreezableVar(null)
 
     @GradleOption(DefaultValues.BooleanTrueDefault::class)
-    @Argument(value = "-no-stdlib", description = "Don't use bundled Kotlin stdlib")
+    @Argument(value = "-no-stdlib", description = "Don't use the default Kotlin/JS stdlib as a compilation dependency")
```

Comment:
It would be better to use the same wording for both JS and JVM.

Reply:
cc @udalov

Reply:
Can you clarify which wording do you think works better and how to apply it to the other platform?

I personally like the version with "Exclude ..." and file paths on JVM more

Reply:
I've asked our proofreader David Watson, he suggests using "Don't automatically include.." for such cases.

Reply:
:okay:

Reply:
LGTM